test-requirements.txt

```diff
@@ -14,3 +14,5 @@ testrepository>=0.0.18
 testscenarios>=0.4
 testtools>=1.4.0
 testresources>=0.2.4
+ipaddress
+datadog
```

Comment:
I'll change the code, that those requirements won't be there anymore

Reply:
datadog is not really needed, but how do you want to replace ipaddress?

Reply:
ipaddress is only used in my changes to neutron.
I've changed it to use netaddr instead.